CMakeLists.txt

```diff
@@ -3,6 +3,12 @@ project(WebAssemblyInterface)
 
 set(CMAKE_CXX_STANDARD 17)
 
+if(${CMAKE_VERSION} VERSION_LESS 3.24)
+  set(download_extract_timestamp_flag)
+else()
+  set(download_extract_timestamp_flag DOWNLOAD_EXTRACT_TIMESTAMP TRUE)
+endif()
```

Comment:
Not sure if this is the appropriate place within the file.

Reply:
Resolving then.

Reply:
Looks like it did not work:
https://github.com/InsightSoftwareConsortium/ITKElastix/pull/248#discussion_r1329355073